docker/arbitrum/arbitrum.sh

```diff
@@ -77,26 +84,30 @@ create_deployment_config_file() {
     INBOX=\"$(echo $INBOX | python -c 'import eth_utils; print(eth_utils.to_checksum_address(input()))')\"
 
     sed "s/\${l1_messenger_args}/${BRIDGE}, ${INBOX}/" \
-        ${ROOT}/scripts/deployment/arbitrum-local-template.json \
+        ${ROOT}/deployments/config/local/412346-arbitrum.json \
         > ${DEPLOYMENT_CONFIG_FILE}
 }
 
 e2e_test() {
+    image_id=$(docker ps --filter "ancestor=nitro-testnode-scripts" --format "{{.ID}}")
+    if [[ ! -z $image_id ]]; then
+        docker stop $image_id
+    fi
     l2_rpc=http://0.0.0.0:8547
     password=""
-    l1_messenger=$(cat ${DEPLOYMENT_DIR}/deployment.json | jq -r '.base_chain.ArbitrumL1Messenger.address')
+    l1_messenger=$(cat ${ARTIFACT_DIR}/412346-arbitrum.deployment.json | jq -r '.base.ArbitrumL1Messenger.address')
 
     network_config="${CACHE_DIR}/localnetwork.json"
     echo Copying network config to $network_config
     docker-compose -f ${NITRO}/docker-compose.yaml run --entrypoint sh testnode-tokenbridge \
                    -c "cat localNetwork.json" > $network_config
 
-    e2e_test_fill ${DEPLOYMENT_DIR} ${KEYFILE} "${password}" $l2_rpc
+    e2e_test_fill $ARTIFACT_DIR $KEYFILE "${password}" $l2_rpc $ABI_DIR
 
     export ARBITRUM_L1_MESSENGER=$l1_messenger
 
     e2e_test_relayer http://0.0.0.0:8545 $l2_rpc $network_config $PRIVKEY $e2e_test_l2_txhash
-    e2e_test_verify ${DEPLOYMENT_DIR} $l2_rpc $ADDRESS $e2e_test_request_id
+    e2e_test_verify $ARTIFACT_DIR $l2_rpc $ADDRESS $e2e_test_request_id $ABI_DIR
```

Comment:
I notice that `e2e_test_fill` and `e2e_test_verify` share some of the arguments, however, they are in different positions. Could we reposition the arguments, so that the shared ones come first, in the same positions.
Something like
```
e2e_test_fill $ARTIFACT_DIR $ABI_DIR $l2_rpc $KEYFILE "${password}" 
e2e_test_verify $ARTIFACT_DIR $ABI_DIR $l2_rpc $ADDRESS $e2e_test_request_id
```
?